.env.example

```
@@ -1,3 +1,7 @@
 OPENCAGEDATA_API_KEY=
 GEOAPIFY_API_KEY=
+SECRET_KEY=
```

```suggestion
FLASK_SECRET_KEY=
```
nit; more descriptive name